### PR TITLE
Fixing relaunch

### PR DESCRIPTION
**Pull request by uffemcev**

### CODE REVIEW

File: winget-install.ps1 [hunk 1]
```diff
@@ -907,17 +907,16 @@ if (Get-WingetStatus) {
...
-    Write-Output "-ForceClose is specified, relaunching in conhost in 10 seconds..."
-    Start-Sleep -Seconds 10
-
-    if ($currentProcessModuleName -ne "conhost") {
-        # Remove -ForceClose from the command line, case-insensitive
-        $newInvocationLine = $MyInvocation.Line -replace '(?i)-ForceClose', ''
+    Write-Output "ForceClose parameter is specified. Conflicting processes will be closed automatically!"
+    if ($currentProcessModuleName -eq "WindowsTerminal") {
+        Write-Output "Terminal detected, relaunching in conhost in 10 seconds..."
+        Write-Output "It may breaks your custom batch files and ps1 scripts with extra commands!"
```
Comment (asheroto) [inline winget-install.ps1:915]: This should say "It may break your custom..."

**Review by asheroto — COMMENTED** [review]
I think this looks good! Thanks again.

I believe only one change to make and then we can approve.

**Review by asheroto — CHANGES_REQUESTED** [review]
Changing status to `request changes`

File: winget-install.ps1 (hunk 1 above)
Comment (uffemcev) [reply winget-install.ps1]: Done! :)

**Comment by uffemcev** [comment]
So, can you approve this now?

**Comment by asheroto** [comment]
Looks great, thank you again. Sorry for the delay, I've been busy with my day job. 😊

**Pull request merged**